iocore/dns/SplitDNS.cc

```diff
@@ -407,7 +407,7 @@ SplitDNSRecord::ProcessDomainSrchList(char *val)
 
   for (int i = 0; i < numTok; i++) {
     const char *current = pTok[i];
-    int cnt             = sz += strlen(current);
+    int cnt = sz += strlen(current);
```

Comment:
This is an interesting change. Most of the other changes add alignment with the previous `=` or whatever, this removes it. (This isn't a problem, just something that caught my eye.)

Reply:
Yeh, there are still imperfections :).